slides/developer-guide/working-with-themes/get-a-slide-format-scheme-from-a-powerpoint-presentation/_index.md

```diff
@@ -0,0 +1,458 @@
+---
+title: "Get a Slide Format Scheme from a PowerPoint Presentation"
+type: docs
+url: /get-a-slide-format-scheme-from-a-powerpoint-presentation/
+weight: 20
+---
+
+## **Introduction**
+
+With Aspose.Slides Cloud, you can easily read information about a format scheme specified to slide elements in a PowerPoint presentation. The API method below allows you to get the following data specifying a slide's major appearance defined in a design theme:
```

Comment:
**specified to slide elements** is bad
**specified for slide elements** or **applied to slide elements**